### PR TITLE
{lib}[GCCcore/6.4.0] libtool v2.4.6

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2570.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bc3335170cea51be70d8fd541143b2db for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca014.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4f2c79983229c986117edafa8ab57447 for a full test report.

File: easybuild/easyconfigs/l/libtool/libtool-2.4.6-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e3bd4d5d3d025a36c21dd6af7ea818a2afcd4dfc1ea5a17b39d7854bcd0c06e3']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('M4', '1.4.18'),
+]
```
Comment (verdurin) [inline easybuild/easyconfigs/l/libtool/libtool-2.4.6-GCCcore-6.4.0.eb]: Please add sanity check

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/a9b5ee4fb83585331efeea43a0dda168 for a full test report.

**Comment by verdurin** [comment]
Looks good.

**Comment by verdurin** [comment]
Merging, thanks @JackPerdue

**Pull request merged**